openslides/motions/models.py

```diff
@@ -697,6 +698,57 @@ def get_root_rest_element(self):
         return self.motion
 
 
+class MotionChangeRecommendation(RESTModelMixin, models.Model):
+    """
+    A MotionChangeRecommendation object saves change recommendations for a specific MotionVersion
+    """
+
+    access_permissions = MotionChangeRecommendationAccessPermissions()
+
```

Comment:
Should be resolved by https://github.com/OpenSlides/OpenSlides/pull/2501/commits/25a208bab2dbf0ac44dc8c451d3eb28972ce446d